packages/headless/doc-parser/src/actions-resolver.ts

```diff
@@ -0,0 +1,225 @@
+import {
+  ApiEntryPoint,
+  ApiInterface,
+  ApiItem,
+  ApiItemKind,
+  ApiNamespace,
+  ApiVariable,
+} from '@microsoft/api-extractor-model';
+import {DocNode, DocParamBlock} from '@microsoft/tsdoc';
+import {findApi} from './api-finder';
+import {AnyEntity, Entity, ObjEntity} from './entity';
+import {buildObjEntity} from './entity-builder';
+import {resolveInterfaceMembers} from './interface-resolver';
+import {emitAsTsDoc} from './tsdoc-emitter';
+
+interface ActionsNamespace {
+  name: string;
+  members: Action[];
+}
+
+enum ActionKind {
+  Normal = 'normalAction',
+  Analytics = 'analyticsAction',
+}
+
+interface BaseAction {
+  name: string;
+  desc: string;
+  parameter: AnyEntity | null;
+  actionKind: ActionKind;
+}
+
+enum AnalyticsKind {
+  Search = 'search',
+  Click = 'click',
+  Custom = 'custom',
+}
```

Comment:
For now I added this as an attribute for analytics actions.

However, @kaitlynkatz and I had a discussion last week and were thinking we could refactor the namespaces of analytics actions in v1 to reflect these three types (instead of having several namespaces for analytics, we'd have three).

Opinions?